docs/updating_schemas.md

```diff
@@ -181,15 +195,49 @@ This new object is an array of example objects that can successfully POST agains
                     res = testapp.post_json('/user', item)
                     return testapp.get(res.location).json
```

Comment:
Post and patch will usually GET the object after creating it and return it, so this GET is redundant.

Reply:
removed